utility/types/mempool_test.go

```diff
@@ -0,0 +1,55 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMempool(t *testing.T) {
+	type args struct {
+		maxTransactionBytes uint64
+		maxTransactions     uint32
+		initialElements     *[][]byte
+		actions             *[]func(*fIFOMempool)
+	}
+	tests := []struct {
+		name      string
+		args      args
+		wantItems [][]byte
+	}{
+		//TODO: Add test cases.
```

Comment:
I've found that I usually forget what I'd ideally want to test if I had time, and this sort of thing is great as starter tasks or bounty tickets for people getting involved from the community.

```suggestion
		// ADDTEST: Test1
		// ADDTEST: Test2
```